src/Moryx.Runtime.Endpoints/Databases/Endpoint/DatabaseController.cs

```diff
@@ -175,8 +176,8 @@ public ActionResult<InvocationResponse> RestoreDatabase(string targetModel, Rest
             return new InvocationResponse();
         }
 
-        [HttpPost("model/{targetModel}/{migrationName}/migrate")]
-        public ActionResult<DatabaseUpdateSummary> MigrateDatabaseModel(string targetModel, string migrationName, DatabaseConfigModel configModel)
+        [HttpPost("model/{targetModel}/migrate")]
+        public async Task<ActionResult<DatabaseMigrationSummary>> MigrateDatabaseModel(string targetModel, DatabaseConfigModel configModel)
```

Comment:
Since this endpoint is changed, the corresponding UI should also change, I am not sure if that is considered or no

Reply:
AFAIK changing to async task does not affect REST signature. It just syntactic sugar in C#, but in the end it's still HTTP Request<->Response.

Reply:
I am not talking about the async, its the migrationName which is excluded in the new endpoint and the previous UI was making a request with two URL parameters but the new one only needs targetModel. That is for sure a new endpoint

Reply:
Good catch, yes that is breaking.